src/ui/hexedit.cc

```diff
@@ -434,6 +435,7 @@ void HexEdit::paintEvent(QPaintEvent *event) {
   painter.drawText(startMargin_ - startPosX_,
                    separatorLength - horizontalAreaSpaceWidth_,
                    statusBarText());
```

Comment:
Overall I'm not a big fan of leaving unused code like this lying around - if someone would like to use it again later on then he could simply get it from git history. This introduces some unnecessary complications - like unused methods i.e. statusBarText - without checking that it now isn't called anywhere one could be surprised that editing it doesn't have any effect.